src/main.ts

```diff
@@ -34,5 +34,6 @@ if (isNodeProd()) {
 	});
 } else {
 	// Dev/test single process, don't need clustering
+	// eslint-disable-next-line @typescript-eslint/no-floating-promises
 	start();
 }
```

Comment:
Why to disable the rule? can't we use `await` here?

Reply:
We could, but there's no need to keep the reference since we're not doing anything with it, we just want it to start

Reply:
I've seen `await` at many places just cuz of readability and  doing nothing with reference.

Reply:
That's because we want those functions to actually wait, in this case in we don't care about waiting.  Also, we can't use await outside an async function, and there would be no point of putting it an async function because you still need to call it outside of it and ignore the promise.